a.core_concepts.md

```diff
@@ -77,9 +77,9 @@ kubectl run nginx --image=nginx --restart=Never --dry-run=client -o yaml | kubec
 <p>
 
 ```bash
-kubectl run busybox --image=busybox --command --restart=Never -it -- env # -it will help in seeing the output
```

Comment:
Default is `restart=Always`, which implies that the busybox container will be restarted indefinitely...
Pod lifecyle in this case: 
> start container > execute command > stop container > restart container >  ... (loop infinitely...)

Same comment is valid for several similar changes in this PR.

Reply:
@StevenJDH any thoughts about that?

Reply:
@dgkanatsios, my mistake, merlixo makes a good point. I was thinking about something else when I made these changes since I normally use busybox with something like `sleep 3600` to keep it running for an hour in labs, or to use it as a temp container using `--rm` to run some tests. 

In other words, how it was before my changes is fine as long as it makes sense to leave pods behind with a Completed status since `--restart=Never` is used. Cleaning that up could be good practice, but perhaps modifying it to include `--rm` makes more sense since we are only using the pod to output environment variables in this case, and having that self delete after the output is good too, for example, `kubectl run busybox --image=busybox --command --restart=Never --rm -it -- env`. I'll leave it up to you all to decide.